web-app/src/main/webapp/images/options-menu.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="50" height="50" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
if you got these icons from elsewhere, make sure the license is attached somewhere.